UAS.Client/src/views/Home.vue

```diff
@@ -1,18 +1,24 @@
 <template>
   <div class="home">
-    <img alt="Vue logo" src="../assets/logo.png" />
-    <HelloWorld msg="Welcome to Your Vue.js App" />
+    <TheHeader/>
```

Comment:
IMO missing space ( ͡° ͜ʖ ͡°)
Better that way:
`<TheHeader />`

Reply:
Changes made in commit 0aad636.